types/promise-map-limit/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "no-unnecessary-generics": false
```

Comment:
This file should only `"extends": "dtslint/dt.json"`, you can disable the rule in code like `// tslint:disable-next-line no-unnecessary-generics`